src/libraries/ComparorUint256.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.4.13;
+import "ROOT/libraries/DelegationTarget.sol";
+
+
+contract ComparorUint256 is DelegationTarget {
```

Comment:
Since we no longer have generic implementations of the collections, I think we can get rid of this contract all together and just inline the checks.  The code will be much easier to read I suspect.